apps/webservice/Dockerfile

```diff
@@ -59,6 +60,7 @@ EXPOSE 3000
 ENV PORT=3000
 ENV AUTH_TRUST_HOST=true
 ENV NODE_ENV=production
+ENV NODE_PATH=/app/apps/webservice/node_modules
```

Comment:
Why do we need this @zacharyblasczyk

Reply:
I don't think we need it. Cleaning it up and testing.